easyconfigs/m/modkit/modkit-0.3.1rc1-GCCcore-12.2.0-Rust-1.75.0.eb

```diff
@@ -0,0 +1,627 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'Cargo'
+
+name = 'modkit'
+version = '0.3.1rc1'
+local_rustver = '1.75.0'
+versionsuffix = '-Rust-' + local_rustver
```

Comment:
Looking at these easyconfs, I'm not sure it's necessary to have Rust in a version suffix, particularly as it's only a build dep:

- https://github.com/easybuilders/easybuild-easyconfigs/blob/cce1c8fdc5360d075d8c39cc34afb28b79fe40c4/easybuild/easyconfigs/m/mfqe/mfqe-0.5.0-GCC-12.3.0.eb
- https://github.com/easybuilders/easybuild-easyconfigs/blob/cce1c8fdc5360d075d8c39cc34afb28b79fe40c4/easybuild/easyconfigs/p/pydantic/pydantic-2.5.3-GCCcore-12.2.0.eb